### PR TITLE
Removes aws cni manual steps, operator handles this

### PR DESCRIPTION
**Pull request by mikestephen**
## Description

Removes manual steps for setting Felix's route table range when using AWS CNI.
The operator already takes care of this, so the manual steps are no longer needed.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
